Userland/Libraries/LibWeb/XML/XMLDocumentBuilder.cpp

```diff
@@ -88,10 +88,14 @@ void XMLDocumentBuilder::element_start(const XML::Name& name, HashMap<XML::Name,
 
     for (auto const& attribute : attributes) {
         // https://www.w3.org/TR/2006/REC-xml-names11-20060816/#ns-decl
-        if (attribute.key == "xmlns" || attribute.key.starts_with("xmlns:"sv)) {
+        if (attribute.key == "xmlns" || (attribute.key.contains(":"sv) && !attribute.key.starts_with("xml"sv))) {
             auto name = attribute.key;
-            // The prefix xmlns is used only to declare namespace bindings and is by definition bound to the namespace name http://www.w3.org/2000/xmlns/.
-            MUST(node->set_attribute_ns(Namespace::XMLNS, MUST(FlyString::from_deprecated_fly_string(name)), MUST(String::from_byte_string(attribute.value))));
+            if (name.starts_with("xmlns:"sv) && name != "xmlsns:" && name != "xmlns:xmlns") {
```

Comment:
while we're here, this check could be `name != "foo"sv && name != "bar"sv` (note the extra sv), or it could be `!name.is_one_of("foo"sv, "bar"sv)`

Reply:
Both those don't need to be sv they can be static arrays. I don't know if sv will bring much more overhead. 
It aids in readability for sure!